GNUmakefile

```diff
@@ -7,7 +7,7 @@ BIN_NAME=terraform-provider-genesyscloud
 BIN_PATH=${DIST_DIR}/${BIN_NAME}
 
 PLUGINS_DIR=~/.terraform.d/plugins
-PLUGIN_PATH=genesys.com/mypurecloud/genesyscloud
+PLUGIN_PATH=genesys.local.com/mypurecloud/genesyscloud
```

Comment:
Guessing you didn't mean to include this

Reply:
Fixed